### PR TITLE
[v7r2] expose deleteProxyBundle to ProxyManagerClient

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Framework
CHANGE: expose deleteProxyBundle call to ProxyManagerClient

ENDRELEASENOTES

### CODE REVIEW

File: FrameworkSystem/Client/ProxyManagerClient.py [hunk 1]
```diff
@@ -475,6 +476,16 @@ def deleteGeneratedProxyFile(self, chain):
...
     return S_OK()
 
+  def deleteProxyBundle(self, idList):
+    """ delete a list of id's
+
+        :param list,tuple idList: list of identity numbers
+
+        :return: S_OK(int)/S_ERROR()
+    """
+    rpcClient = RPCClient("Framework/ProxyManager", timeout=120)
```
Comment (chaen) [inline FrameworkSystem/Client/ProxyManagerClient.py:486]: should be `_getRPC` no ?

File: FrameworkSystem/Client/ProxyManagerClient.py [hunk 2]
```diff
@@ -560,15 +571,15 @@ def renewProxy(self, proxyToBeRenewed=None, minLifeTime=3600, newProxyLifeTime=4
 
     return S_OK(chain)
 
-  def getDBContents(self, condDict={}):
+  def getDBContents(self, condDict={}, sort=[['UserDN', 'DESC']], start=0, limit=0):
```
Comment (chaen) [inline FrameworkSystem/Client/ProxyManagerClient.py:574]: avoid maybe `sort` which is reserved ?

File: FrameworkSystem/Client/ProxyManagerClient.py (hunk 1 above)
Comment (fstagni) [reply FrameworkSystem/Client/ProxyManagerClient.py:486]: For a reason unknown to me (a good one? not sure) this `ProxyManagerClient` does not inherit from `Client`, so there's no `_getRPC`). That's why I am exposing this RPC call directly here.

File: FrameworkSystem/Client/ProxyManagerClient.py (hunk 2 above)
Comment (fstagni) [reply FrameworkSystem/Client/ProxyManagerClient.py]: Done.

**Pull request merged**